File: Consonant value/Consonant value.py
import string,re
def solve(s):
    sum_v = lambda x: list(s)
    return max([sum([string.ascii_lowercase.index(tmp2)+1 for tmp2 in list(tmp)])  if len(s)>0 else 0 for tmp in re.split(r"[aiueo]",s)])

File: Consonant value/Consonant value.js
function solve(s) {
  const alpha = "abcdefghijklmnopqrstuvwxyz"
  const sum_c = (s) => s.split("").map(v => alpha.indexOf(v)+1).reduce((a,b) => a+b,0)
  return Math.max.apply(null,s.split(/[aiueo]/).map(v => v.length>0 ? sum_c(v) : 0));
};
